coursedata/quizzes/el.yaml

```diff
@@ -62,16 +62,16 @@ levels:
             question_score: '10'
         10:
             mp_choice_options:
```

Comment:
These texts are generated by Weblate. I guess to minimize conflicts, we want to remove these form the PR and just make the change in En and the actually translated quizzes (such as nl, for example)